crates/rattler_installs_packages/src/artifacts/sdist.rs

```diff
@@ -266,6 +267,7 @@ mod tests {
             None,
             &resolve_options,
             package_db.1.path(),
+            Default::default(),
```

Comment:
According to new linting rules I've heard from @baszalmstra and @wolfv, we now want to be explicit when specifying the default so `MyStruct::default()` instead of `Default::default()`